### DISCUSSION TITLE
filebeat apache module

### DISCUSSION THREAD

**Discussion by superbaim** [Security-Onion-Solutions/securityonion#4527] [Unsupported Versions]
Hi,

Im testing SecurityOnion 2.3.50 with filebeat apache module, all the beats field like agent name os version etc parse correctly but why the field message(the apace access.log) it self doesnt parse ? Is this the default behaviour if using beats ? 

Thanks

**Comment by weslambert** [comment]
It depends on how you have it set up, and honestly, it may be best to wait top troubleshoot more once we officially support Filebeat modules OOB, in 2.3.60:

https://blog.securityonion.net/2021/06/upcoming-security-onion-2360-release.html
